package.json

```diff
@@ -42,7 +42,7 @@
     "test:unit": "nyc mocha --require ts-node/register --reporter spec --slow 200 --bail --timeout 10000 test/socket.io.ts",
     "format:check": "prettier --check \"lib/**/*.ts\" \"test/**/*.ts\"",
     "format:fix": "prettier --write \"lib/**/*.ts\" \"test/**/*.ts\"",
-    "prepack": "npm run compile"
+    "prepare": "npm run compile"
```

Comment:
use prepare instead of prepack so that installing socket.io from git with npm works

The downside of this is that yarn doesn't follow npm spec, and yarn needs prepack instead of prepare, so install from git with yarn will no longer work. Either npm works, or yarn works. I think it's a better choice to support the standard tool for npmjs.com.